### PR TITLE
address additional comments from pr #10259

### PR DESCRIPTION
**Pull request by juanvallejo**
https://github.com/kubernetes/website/pull/10422 merged into 1.12 branch.

Picking into master

cc @soltysh

### CODE REVIEW

File: content/en/docs/tasks/extend-kubectl/kubectl-plugins.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ I am a plugin with a dash in my name
 
 #### Name conflicts and overshadowing
 
-It can be possible to have multiple pluins with the same filename in different locations throughout your PATH.
+It can be possible to have multiple plugins with the same filename in different locations throughout your PATH.
```
Comment (kbhawkey) [inline content/en/docs/tasks/extend-kubectl/kubectl-plugins.md]: It is possible

**Comment by tengqm** [comment]
A rebase is needed to resolve the conflicts, otherwise this one looks good to me.

**Comment by juanvallejo** [comment]
@tengqm Thanks, rebased PR

**Pull request merged**